DEPENDENCIES

```diff
@@ -190,6 +190,7 @@ maven/mavencentral/org.springframework/spring-beans/6.0.7, Apache-2.0, approved,
 maven/mavencentral/org.springframework/spring-context/6.0.7, Apache-2.0, approved, #5936
 maven/mavencentral/org.springframework/spring-core/6.0.7, Apache-2.0 AND BSD-3-Clause, approved, #5948
 maven/mavencentral/org.springframework/spring-expression/6.0.7, Apache-2.0, approved, #3284
```

Comment:
How can this be? Do you now really have both versions in your repo or did you not run the dash tool properly?

Reply:
There was an error with running the tool, you are correct, we only use new version.

Reply:
@adkumar1 @dvasunin @Wulghash make sure to never do this manually somehow! ever again!